### PR TITLE
Clean up: copyright

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

File: schemas/data/copyright.schema.tpl.json [hunk 1]
```diff
@@ -9,15 +9,15 @@
...
+      "_instruction": "Add all parties that hold this copyright.",
       "_linkedCategories": [
          "legalPerson"
       ]
     },
     "year": {
       "type": "string",
       "pattern": "([0-9]{4})",
-      "_instruction": "Enter the year during which the copyright was first asserted."
+      "_instruction": "Enter the year during which the copyright was first asserted follwoing the defined pattern (e.g., 2023)."
```
Comment (lzehl) [inline schemas/data/copyright.schema.tpl.json]: @apdavison this is something we need to overthink correct? in principle the copyright has to be renewed for each year which would make the link to this type an array and then this instruction should read something like this: "Enter the year during which the copyright was claimed following the defined pattern (e.g., 2023)."

@UlrikeS91 besides this issue there is a typo in "follwoing" :wink:

**Review by lzehl — CHANGES_REQUESTED** [review]

File: schemas/data/copyright.schema.tpl.json (hunk 1 above)
Comment (UlrikeS91) [reply schemas/data/copyright.schema.tpl.json]: I'll wait for feedback form @apdavison before making any changes

Comment (apdavison) [reply schemas/data/copyright.schema.tpl.json]: yes, an array would be a nice solution. The instructions should then be something like "Enter the year during which the copyright was first asserted and, optionally, later years during which updated versions were published."

Comment (lzehl) [reply schemas/data/copyright.schema.tpl.json]: perfect. thanks for the feedback. I'll do the changes

**Review by lzehl — APPROVED** [review]
@apdavison I've updated the schema. Please double check and feel free to merge if you approve. Question: should the copyright on RPVs should also become an array? (in case the copyright holder changed for a published version?)

**Review by Peyman-N — APPROVED** [review]

**Pull request merged**